packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol

```diff
@@ -24,10 +24,10 @@ import { Encoding } from "../libraries/Encoding.sol";
 contract CrossDomainMessengerLegacySpacer {
     /**
      * @custom:legacy
-     * @custom:spacer libAddressManager
+     * @custom:spacer address libAddressManager
      * @notice Spacer for backwards compatibility.
      */
-    address internal spacer0;
```

Comment:
How exactly does the naming scheme work? Perhaps add to readme

Reply:
Added to README